usr/src/uts/common/sys/time.h

```diff
@@ -243,6 +247,9 @@ struct itimerval32 {
 #define	MSEC2NSEC(m)	((hrtime_t)(m) * (NANOSEC / MILLISEC))
 #define	NSEC2MSEC(n)	((n) / (NANOSEC / MILLISEC))
 
+#define	NSEC2SEC(n)	((n) / (NANOSEC / SEC))
+#define	SEC2NSEC(m)	((hrtime_t)(m) * (NANOSEC / SEC))
```

Comment:
Please at least file a new bug (and include the bug number in the commit message).  This way it's easier to see that these showed up.  (Personally, I'd even like these to be a separate commit...I think it's easy enough to do, and it makes it easier for folks to backport what they want more easily.  It also allows the advocates to revert one piece without manually having to separate out the changes.)